advanced/am-pattern-1/values.yaml

```diff
@@ -0,0 +1,174 @@
+# Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+wso2:
+
+  # WSO2 subscription parameters. If you do not have an active subscription, do not provide values for the parameters.
+  # If you do not possess an active WSO2 subscription already, you can sign up for a WSO2 Free Trial Subscription at (https://wso2.com/free-trial-subscription).
+  subscription:
+    username: ""
+    password: ""
+
+  deployment:
+    dependencies:
+      # The configuration should be set to be 'true' if a MySQL database should be spawned as a pod within the cluster
+      mysql: true
+      # Enable NFS dynamic provisioner for Kubernetes
+      nfsServerProvisioner: true
+
+    persistentRuntimeArtifacts:
+      # Persistent storage provider expected to be used for sharing persistent runtime artifacts
+      # This can be one of the following.
+      #   1. internal-nfs - Using the NFS dynamic provisioner for Kubernetes
+      #   2. external-nfs - Using an externally configured NFS server
+      cloudProvider: "internal-nfs"
+      # External NFS Server IP
+      nfsServerIP: "xxxxxxxxxxx"
```

Comment:
@Aaquiff leave `nfsServerIP`, `sharedAPIMSynapseConfigsPath` and `sharedAPIMExecutionPlansPath` empty, as this would fail the logic during the selection of internal and external NFS.

Reply:
Fixed